manager/integration/tests/test_settings.py

```diff
@@ -788,3 +793,260 @@ def test_setting_concurrent_rebuild_limit():  # NOQA
     (Not sure if we need to put this specific case as a separate test in
      test_engine_upgrade.py)
```

Comment:
Probably need to make this a separate test case.

Reply:
HI @shuo-wu ,

I remove this part to case `test_engine_live_upgrade_while_replica_concurrent_rebuild` in `test_engine_upgrade.py`